dae/dae/enrichment_tool/enrichment_helper.py

```diff
@@ -57,18 +57,43 @@ def has_enrichment_config(genotype_data: GenotypeData) -> bool:
             .get_enrichment_config(genotype_data) is not None
 
     @staticmethod
-    def get_default_background_model(genotype_data: GenotypeData) -> str:
+    def get_default_background_model(genotype_data: GenotypeData) -> str:  # pylint: disable=missing-function-docstring
         enrichment_config = EnrichmentHelper.get_enrichment_config(
             genotype_data)
         assert enrichment_config is not None
-        return cast(str, enrichment_config["default_background_model"])
+
+        if enrichment_config["default_background_model"]:
+            return enrichment_config["default_background_model"]
+        return enrichment_config["selected_background_models"][0]
 
     @staticmethod
     def get_default_counting_model(genotype_data: GenotypeData) -> str:
         enrichment_config = EnrichmentHelper.get_enrichment_config(
             genotype_data)
         assert enrichment_config is not None
-        return cast(str, enrichment_config["default_counting_model"])
+        return enrichment_config["default_counting_model"]
+
+    @staticmethod
+    def get_selected_counting_models(genotype_data: GenotypeData) -> list[str]:  # pylint: disable=missing-function-docstring
+        enrichment_config = EnrichmentHelper.get_enrichment_config(
+            genotype_data)
+        assert enrichment_config is not None
+
+        if enrichment_config["selected_counting_models"]:
+            return enrichment_config["selected_counting_models"]
+        if enrichment_config["counting"]:
+            return list(enrichment_config["counting"].keys())
+        return ["enrichment_events_counting", "enrichment_gene_counting"]
```

Comment:
remove this after adding default values for "counting" in the enrichment config schema